src/store/modules/voting.js

```diff
@@ -7,12 +7,15 @@ export default {
     apiUrl: 'https://api.willy.radio/2.4',
     listId: 277,
     editionId: 4199,
+    seed: Math.floor(Math.random() * 100 + 1),
```

Comment:
Dit kan zwaar worden mss het minuut een random kiezen, zodat meeste request dezelfde zullen zijn en kunnen gecached worden?
```suggestion
    seed: (new Date()).setSeconds(0,0),
```